app/middleware/defaultError.js

```diff
@@ -0,0 +1,15 @@
+const notFoundError = (req, res) => {
+  res.status(404).json({ error: 'not found' });
+};
+
+// Express recongizes the error handler by the number of arguments
+// the function accepts, so `next` should be defined even if it is
+// not used inside the handler. ESLint will complain about unused
+// vars, we will disable it by no-unused-vars rule.
+/* eslint-disable no-unused-vars */
```

Comment:
You point out a general issue with eslint and express here, so I think I will move you're explanation to the ESLint config and add a more general rule there :)